ui/App/DiagnosticsScreen/Json.svelte

```diff
@@ -0,0 +1,44 @@
+<!--
+ Copyright © 2021 The Radicle Upstream Contributors
+
+ This file is part of radicle-upstream, distributed under the GPLv3
+ with Radicle Linking Exception. For full terms see the included
+ LICENSE file.
+-->
+<script lang="ts">
+  export let data: unknown;
+  export let title: string | undefined = undefined;
+</script>
+
+<style>
+  .container {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    flex: 1;
+    overflow: scroll;
```

Comment:
I don’t think this is necessary. Scrolling happens in the `pre` element when the JSON is too big.

Reply:
See: https://github.com/radicle-dev/radicle-upstream/pull/2566#discussion_r741841058.